fbw-a32nx/src/systems/fmgc/src/guidance/GuidanceController.ts

```diff
@@ -241,7 +242,7 @@ export class GuidanceController {
       // Don't compute distance and ETA for XM legs
       const isXMLeg = activeLeg instanceof FMLeg || activeLeg instanceof VMLeg;
       const efisDistance = isXMLeg ? -1 : Avionics.Utils.computeGreatCircleDistance(ppos, termination);
-      const efisEta = isXMLeg || !etaComputable ? -1 : LnavDriver.legEta(ppos, gs, termination);
+      const efisEta = isXMLeg || !etaComputable ? -1 : LnavDriver.legEta(ppos, gs, termination, this.acConfig);
```

Comment:
It's cleaner to make this method not static IMO.

Reply:
done